From: Josselin Mouette <joss@debian.org>
Date: Sat, 04 Jun 2011 21:07:43 +0000
Subject: Move Preferences and Administration to Settings submenus

Move the old Preferences and Administration settings to
submenus in the “system” submenu. This is merely so that
they have a place until we clean them up entirely.

---
 layout/gnome-applications.menu | 95 ++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 95 insertions(+)

--- a/layout/gnome-applications.menu
+++ b/layout/gnome-applications.menu
@@ -189,6 +189,101 @@
       <Filename>org.gnome.Terminal.desktop</Filename>
       <Filename>org.gnome.Usage.desktop</Filename>
     </Exclude>
+
+    <Menu>
+      <Name>Preferences</Name>
+      <Directory>Settings.directory</Directory>
+      <Include>
+        <And>
+          <Category>Settings</Category>
+          <Not>
+            <Or>
+              <Category>System</Category>
+              <Category>X-GNOME-Settings-Panel</Category>
+              <Filename>authconfig.desktop</Filename>
+              <Filename>ca.desrt.dconf-editor.desktop</Filename>
+              <Filename>fedora-release-notes.desktop</Filename>
+              <Filename>firewall-config.desktop</Filename>
+              <Filename>flash-player-properties.desktop</Filename>
+              <Filename>gconf-editor.desktop</Filename>
+              <Filename>gnome-abrt.desktop</Filename>
+              <Filename>ibus-setup-anthy.desktop</Filename>
+              <Filename>ibus-setup.desktop</Filename>
+              <Filename>ibus-setup-hangul.desktop</Filename>
+              <Filename>ibus-setup-libbopomofo.desktop</Filename>
+              <Filename>ibus-setup-libpinyin.desktop</Filename>
+              <Filename>ibus-setup-m17n.desktop</Filename>
+              <Filename>ibus-setup-typing-booster.desktop</Filename>
+              <Filename>im-chooser.desktop</Filename>
+              <Filename>itweb-settings.desktop</Filename>
+              <Filename>jhbuild.desktop</Filename>
+              <Filename>javaws.desktop</Filename>
+              <Filename>java-1.7.0-openjdk-jconsole.desktop</Filename>
+              <Filename>java-1.7.0-openjdk-policytool.desktop</Filename>
+              <Filename>log4j-chainsaw.desktop</Filename>
+              <Filename>log4j-logfactor5.desktop</Filename>
+              <Filename>nm-connection-editor.desktop</Filename>
+              <Filename>org.gnome.PowerStats.desktop</Filename>
+              <Filename>setroubleshoot.desktop</Filename>
+              <Filename>system-config-date.desktop</Filename>
+              <Filename>system-config-firewall.desktop</Filename>
+              <Filename>system-config-keyboard.desktop</Filename>
+              <Filename>system-config-language.desktop</Filename>
+              <Filename>system-config-printer.desktop</Filename>
+              <Filename>system-config-users.desktop</Filename>
+              <Filename>vino-preferences.desktop</Filename>
+            </Or>
+          </Not>
+        </And>
+      </Include>
+    </Menu>
+    <Menu>
+      <Name>Administration</Name>
+      <Directory>Settings-System.directory</Directory>
+      <Include>
+        <And>
+          <Category>Settings</Category>
+          <Category>System</Category>
+          <Not>
+            <Or>
+              <Category>X-GNOME-Settings-Panel</Category>
+              <Filename>authconfig.desktop</Filename>
+              <Filename>ca.desrt.dconf-editor.desktop</Filename>
+              <Filename>fedora-release-notes.desktop</Filename>
+              <Filename>firewall-config.desktop</Filename>
+              <Filename>flash-player-properties.desktop</Filename>
+              <Filename>gconf-editor.desktop</Filename>
+              <Filename>gnome-abrt.desktop</Filename>
+              <Filename>ibus-setup-anthy.desktop</Filename>
+              <Filename>ibus-setup.desktop</Filename>
+              <Filename>ibus-setup-hangul.desktop</Filename>
+              <Filename>ibus-setup-libbopomofo.desktop</Filename>
+              <Filename>ibus-setup-libpinyin.desktop</Filename>
+              <Filename>ibus-setup-m17n.desktop</Filename>
+              <Filename>ibus-setup-typing-booster.desktop</Filename>
+              <Filename>im-chooser.desktop</Filename>
+              <Filename>itweb-settings.desktop</Filename>
+              <Filename>jhbuild.desktop</Filename>
+              <Filename>javaws.desktop</Filename>
+              <Filename>java-1.7.0-openjdk-jconsole.desktop</Filename>
+              <Filename>java-1.7.0-openjdk-policytool.desktop</Filename>
+              <Filename>log4j-chainsaw.desktop</Filename>
+              <Filename>log4j-logfactor5.desktop</Filename>
+              <Filename>nm-connection-editor.desktop</Filename>
+              <Filename>org.gnome.PowerStats.desktop</Filename>
+              <Filename>setroubleshoot.desktop</Filename>
+              <Filename>system-config-date.desktop</Filename>
+              <Filename>system-config-firewall.desktop</Filename>
+              <Filename>system-config-keyboard.desktop</Filename>
+              <Filename>system-config-language.desktop</Filename>
+              <Filename>system-config-printer.desktop</Filename>
+              <Filename>system-config-users.desktop</Filename>
+              <Filename>vino-preferences.desktop</Filename>
+            </Or>
+          </Not>
+        </And>
+      </Include>
+    </Menu>
   </Menu>   <!-- End System Tools -->
 
   <!-- System Settings -->
